.detoxrc.json

```diff
@@ -0,0 +1,51 @@
+{
+  "testRunner": "jest",
+  "runnerConfig": "e2e/config.json",
+  "skipLegacyWorkersInjection": true,
+  "apps": {
+    "ios.debug": {
+      "type": "ios.app",
+      "binaryPath": "ios/build/Build/Products/Debug-iphonesimulator/iNaturalistReactNative.app",
+      "build": "xcodebuild -workspace ios/iNaturalistReactNative.xcworkspace -scheme iNaturalistReactNative -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build"
+    },
+    "ios.release": {
+      "type": "ios.app",
+      "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/iNaturalistReactNative.app",
+      "build": "xcodebuild -workspace ios/iNaturalistReactNative.xcworkspace -scheme iNaturalistReactNative -configuration Release -sdk iphonesimulator -derivedDataPath ios/build"
+    },
+    "android.release": {
+      "type": "android.apk",
+      "binaryPath": "android/app/build/outputs/apk/release/com.inaturalistreactnative-v0.1.1+32-release.apk",
+      "testBinaryPath": "android/app/build/outputs/apk/androidTest/release/com.inaturalistreactnative-v0.1.1+32-release-androidTest.apk",
+      "build": "cd android && ./gradlew assembleRelease assembleAndroidTest -DtestBuildType=release && cd .." 
```

Comment:
Really? It does not work for me without the assembleAndroidTest, and as far as I understood now it should not work. Because the test apk contains the actual UI tests (in Espresso = native Android e2e testing library) that interacts with the release build of our app. So, I think we need both commands and apks. What kind of error did you get when building with assembleAndroidTest?